### ISSUE TITLE
Feature request: Support for Django form validation

### ISSUE THREAD

**Issue by 2malh** [strawberry-graphql/strawberry-graphql-django#23]
Hey,
is there any possibility to add some support for the Django form validation by also returning the field parameters via GraphQL? What I was thinking of is something like this which takes whatever is defined in the model and passes it to the frontend so that I'd be possible to create some auto-validation based on the backend.

```json
User {
  "firstname": {
    "value": "James",
    "validation": {
      "type": "String",
      "max_length": 30
    }
  }
}
```

**Comment by la4de** [comment]
We do not support Django forms yet. That's very interesting idea. One option would be to provide validation information as a separate object.

We are currently designing the API and stablizing it but Django forms are definitely something we want to think about seriously in the near future.

Thanks for your feedback!

**Comment by joeydebreuk** [comment]
Correct me if I'm wrong, but this has little to do with Django Forms? I think what you are looking for is adding meta information to a type that includes a validation schema. I think this would be based on models, not on forms?

**Comment by 2malh** [comment]
> Correct me if I'm wrong, but this has little to do with Django Forms? I think what you are looking for is adding meta information to a type that includes a validation schema. I think this would be based on models, not on forms?

Yes, you're righht. I should've been more clear. What I mean is to expose the model parameters / meta information to the frontend. On top of this I think a CSRF token validation would come in handy.
